test/Table_Tests/src/Common_Table_Operations/Numeric_Column_Spec.enso

```diff
@@ -41,19 +41,19 @@ add_numeric_columns_specs suite_builder setup =
         group_builder.specify "a numeric column supports abs operation" <|
             column = numeric_column.get
             abs_column = (column:Numeric_Column).abs
-            (abs_column : Column|DB_Column).to_vector . should_equal [10, 0, 1, 12]
```

Comment:
@JaroslavTulach you were not happy about `DB_Column` being mentioned in the core Table tests. I have now removed this as _it started causing real trouble_.

Full discussion at https://github.com/enso-org/enso/pull/12990/files#r2066986953